wdl/pipelines/TechAgnostic/Visualization/SvQCPlots.wdl

```diff
@@ -0,0 +1,451 @@
+            { grep -m 1 '^sample' ${caller}_all_sample_stats_with_cov; grep -v '^sample' ${caller}_all_sample_stats_with_cov; } > ${caller}_all_sample_stats_with_cov.txt
+        done
+
+    >>>
+
+    output{
+        Array[File] all_stats_with_cov = glob("*_all_sample_stats_with_cov.txt")
+    }
+
+    #########################
+    RuntimeAttr default_attr = object {
+        cpu_cores:          2,
+        mem_gb:             8,
+        disk_gb:            disk_size,
+        boot_disk_gb:       10,
+        preemptible_tries:  3,
+        max_retries:        0,
+        docker:             "us.gcr.io/broad-dsp-lrma/lr-basic:latest"
+    }
+    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+    runtime {
+        cpu:                    select_first([runtime_attr.cpu_cores,         default_attr.cpu_cores])
+        memory:                 select_first([runtime_attr.mem_gb,            default_attr.mem_gb]) + " GiB"
+        disks: "local-disk " +  select_first([runtime_attr.disk_gb,           default_attr.disk_gb]) + " HDD"
+        bootDiskSizeGb:         select_first([runtime_attr.boot_disk_gb,      default_attr.boot_disk_gb])
+        preemptible:            select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
+        maxRetries:             select_first([runtime_attr.max_retries,       default_attr.max_retries])
+        docker:                 select_first([runtime_attr.docker,            default_attr.docker])
+    }
+}
+
+task plotSVQCMetrics{
+
+    parameter_meta{
+        all_stats_with_cov: "List of files containing SV summary stats with coverage"
+        all_stats_by_type: "List of files containing SV stats by type"
+        callers: "List of SV callers used to generate input vcf files"
+        reference_name: "Reference genome name"
+    }
+
+    input{
+        Array[File] all_stats_with_cov
+        Array[File] all_stats_by_type
+        Array[String] callers
+        String reference_name
+        RuntimeAttr? runtime_attr_override
+    }
+    Array[File] input_files = flatten([all_stats_with_cov, all_stats_by_type])
+
+    Int minimal_disk_size = (ceil(size(input_files, "GB")  ) + 100 ) # 100GB buffer
+    Int disk_size = if minimal_disk_size > 100 then minimal_disk_size else 100
+
+    command{
+        set -euo pipefail
+
+        echo "Making directory for input files"
+        mkdir ~{reference_name}
+        echo "Current Directory"
+        ls
+        echo "Moving input files to ~{reference_name}"
+        mv ~{sep=" " input_files} ~{reference_name}/
+        echo "Current Directory After Moving Files"
+        ls
+
+        echo "Files in ~{reference_name} directory:"
+        ls ~{reference_name}
+
+        echo "Running jupyter notebook"
+        papermill /plot_single_sample_stats.ipynb out_plot_single_sample_stats.ipynb \
+        -p reference_in ~{reference_name}  \
+        -p callers_in "~{sep="," callers}"
+    }
+    output{
```

Comment:
minor: allow users to customize the name of the output notebook.

Reply:
done